workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -46,11 +46,68 @@ amenities:
   filter: amenity=* and (type:way or type:node)
 
 poi:
-  name: Points-of-Interests
+  name: Points of interest
+  description: All features with the amenities key.
```

Comment:
This can be misunderstood, because you use more than just the `amanity` key.  Maybe you meant `Amenities of public interest`?